### ISSUE TITLE
Flight master Khaelyn Steelwing at Eastern Plaguelands doesn't open flight map upon talking

### ISSUE THREAD

**Issue by tribaljet** [HellscreamWoW/Tracker#1511]
As the title says, and similar to a Mudsprocket issue, the flight master doesn't open the flight destination map, preventing any outwards flight to happen.

**Comment by GeneralCaveman** [comment]
https://github.com/HellscreamWoW/Tracker/issues/684

**Comment by rist9** [comment]
Yep. Still bugged. Won't even let you continue a flight designated to somewhere else.
